.github/workflows/quality-gates.yml

```
@@ -0,0 +1,278 @@
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build Production
+        run: npm run build
+
+      - name: 🚀 Lighthouse CI
+        uses: treosh/lighthouse-ci-action@v9
```

actionlint (1.7.7)

the runner of "treosh/lighthouse-ci-action@v9" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)